common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -4697,10 +4697,10 @@ object CelebornConf extends Logging {
       .intConf
       .createWithDefault(0)
 
-  val MASTER_SEND_APPLICATION_META_THREADS: ConfigEntry[Int] =
-    buildConf("celeborn.master.send.applicationMeta.threads")
+  val MASTER_SEND_APPLICATION_AUTH_META_THREADS: ConfigEntry[Int] =
+    buildConf("celeborn.master.send.applicationAuthMeta.threads")
       .categories("master")
-      .doc("Number of threads used by the Master to send ApplicationMeta to Workers.")
+      .doc("Number of threads used by the Master to send ApplicationAuthMeta to Workers.")
```

Comment:
I replaced the  case sensitive keywords in IDEA.

`ApplicationMeta` -> `ApplicationAuthMeta` 
`applicationMeta` -> `applicationAuthMeta` 

`APPLICATION_META` -> `APPLICATION_AUTH_META`

Reply:
cc @RexXiong @otterc